.env

```diff
@@ -1,5 +1,5 @@
 OPENID_GIT_URL=https://gitlab.com/openid/conformance-suite.git
-OPENID_GIT_TAG=${OPENID_GIT_TAG:-release-v4.1.11}
+OPENID_GIT_TAG=${OPENID_GIT_TAG:-release-v4.1.12}
```

Comment:
IMO, we should use the latest (4.1.13) version available - https://gitlab.com/openid/conformance-suite/-/tags/release-v4.1.13

Reply:
@VinodAnandan Good point! I will update the version once I check this setup is working with 4.1.13. I will probably do it aftern Keycloak 14 release, so I am able to try everything together before this is merged (Keycloak 14 release together with 4.1.13 testsuite)

Reply:
@VinodAnandan @tnorimat Updated to 4.1.13. Also updated Keycloak to 14 in this PR. Thanks!